src/main/java/no/nav/tag/finnkandidatapi/metrikker/sensu/SensuClient.java

```diff
@@ -0,0 +1,130 @@
+package no.nav.tag.finnkandidatapi.metrikker.sensu;
+
+import com.google.common.base.Joiner;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.net.*;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+@Slf4j
+public class SensuClient {
+
+    private String environmentName;
+    private final String sensuHost;
+    private int sensuPort;
+    private String hostName;
+    private String applicationName;
+
+    public SensuClient(
+            String environmentName,
+            String sensuHostname,
+            int sensuPort,
+            String applicationName
+    ) {
+        this.environmentName = environmentName;
+        this.sensuHost = sensuHostname;
+        this.sensuPort = sensuPort;
+        this.applicationName = applicationName;
+        this.hostName = getHostname();
+    }
+
+    public void sendEvent(String event) {
+        sendEvent(event, Collections.emptyMap(), Map.of("value", 1));
+    }
+
+    public void sendEvent(String event, Map fields) {
+        sendEvent(event, Collections.emptyMap(), fields);
+    }
+
+    @SuppressWarnings({ "rawtypes", "unchecked" })
+    public void sendEvent(String event, Map tags, Map fields) {
```

Comment:
Tags blir ideksert i InfluxDB. Så man bruker tags for å kunne gjøre spørringer i Grafana (typ de i WHERE clause). Verdiene man sender med i fields kan man hente ut i Grafana og vise dem i en visualisering